README.md

```diff
@@ -122,13 +122,21 @@ $ node example.js | pino-pretty
 
 const http = require('http')
 const server = http.createServer(handle)
+const { v4: uuidv4 } = require('uuid')
```

Comment:
Could you recommend https://nodejs.org/api/crypto.html#cryptorandomuuidoptions instead? It's _faster_ and does not require external deps.

Reply:
absolutely, thanks for the suggestion, this is done